ci/rebase_analysis.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
```

Comment:
I'm updating it to use `#!/usr/bin/env python2` with next force push. Let me know if that isn't right.